python/cudf/cudf/core/column/numerical.py

```diff
@@ -573,20 +573,6 @@ def _normalize_find_and_replace_input(input_column_dtype, col_to_normalize):
     return normalized_column.astype(input_column_dtype)
 
 
-def column_hash_values(column0, *other_columns, initial_hash_values=None):
```

Comment:
Is it ok to have removed this, or should I update this to use the new API?

Reply:
It should be okay to remove, I believe this was just being used underneath other functions